bin/fuel-core/src/cli/run.rs

```diff
@@ -207,6 +207,11 @@ pub struct Command {
     #[arg(long = "min-da-gas-price", default_value = "10000000", env)]
     pub min_da_gas_price: u64,
 
+    /// Maximum DA gas price
+    /// DEV: ensure that the max_da_gas_price default is > then the min_da_gas_price default
```

Comment:
```suggestion
    // DEV: ensure that the max_da_gas_price default is > then the min_da_gas_price default
```

Should this be part of the cli help text? If so, I don't get what "DEV: " is for.

Reply:
err no it shouldn't, it's just a note for developers modifying the limits